manifests/FlorianBalmer/Notepad2/4.2.25.yaml

```diff
@@ -0,0 +1,20 @@
+Id: flos-freeware.Notepad2
```

Comment:
```suggestion
Id: FlorianBalmer.Notepad2
```
I think you may try to change the value of `Id` to `FlorianBalmer.Notepad2` in order to pass the validation test.